spec/src/main/doc/javase/scopescontext_se.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[contexts_se]]
+
+== Scopes and contexts in Java SE
+
+[[builtin_contexts_se]]
+
+=== Context management for built-in scopes in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<builtin_contexts>> and is also required to ensure the following rules for built-in context implementation.
+
+
+[[request_context_se]]
+
+==== Request context lifecycle in Java SE
+
+When running in Java SE, the container must extend the rules defined in <<request_context>> and is also required to ensure the following rules.
+
+The request scope is active:
+
+* during any method invocation.
```

Comment:
OK, so my understanding is that you haven't really thought about this. You just put it here vaguely for us to figure out and make work somehow. And then it will be changed anyway. Correct?

In that case my suggestion would be to figure out the semantics (not necessarily the implementation) first and only then to put it into the spec.